mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1064,8 +1066,7 @@ func resourceContainerCluster() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: schemaNodePool,
 				},
-				Description: `List of node pools associated with this cluster. See google_container_node_pool for schema. Warning: node pools defined inside a cluster can't be changed (or added/removed) after cluster creation without deleting and recreating the entire cluster. Unless you absolutely need the ability to say "these are the only node pools associated with this cluster", use the google_container_node_pool resource instead of this property.`,
-				ConflictsWith: []string{"enable_autopilot"},
```

Comment:
This is not really how we have to do this.  The right field to use is cluster_autoscaling.auto_provisioning_defaults.service_account

Reply:
Ok, fair enough. Closing this PR in the hopes of https://github.com/GoogleCloudPlatform/magic-modules/pull/6733 being merged eventually